packages/javascript-sdk/src/oauth2-client/index.ts

```diff
@@ -185,6 +186,7 @@ abstract class OAuth2Client {
       idToken: responseObject.id_token,
       refreshToken: responseObject.refresh_token,
       tokenExpiry: tokenExpiry,
+      rawResponse: await responseClone.text(),
```

Comment:
so the basic ask is to just include the entire response object in addition to how we structure it normally.

i'm curious if we should include this as an _opt in_ configuration? it may be overkill to add as a config but i also think we are adding a lot to what was previously a very thin api that could add noise for many who just don't need this info